graph/resolver/posts_connection.resolvers.go

```diff
@@ -5,19 +5,12 @@ package resolver
 
 import (
 	"context"
-	"fmt"
 
-	"github.com/nedrocks/delphisbe/graph/generated"
 	"github.com/nedrocks/delphisbe/graph/model"
 )
 
 func (r *postsConnectionResolver) Edges(ctx context.Context, obj *model.PostsConnection) ([]*model.PostsEdge, error) {
-	panic(fmt.Errorf("not implemented"))
-}
-
-// PostsConnection returns generated.PostsConnectionResolver implementation.
-func (r *Resolver) PostsConnection() generated.PostsConnectionResolver {
-	return &postsConnectionResolver{r}
+	return obj.Edges, nil
```

Comment:
No need of fetching Edges here, it is done at datastore level.

Reply:
This is okay for now, but we should be careful here because you could get to `Edges` without having resolved the edges in some cases. The awesome thing about GQL is once you implement a resolver it "just works" but the gotcha is you can get to a resolver in a number of ways.

We will need to be careful for when we can get to this connection in another way (for instance maybe I want to see all of the Posts I've created...)

Reply:
Yeah this makes sense. We'll figure out a better strategy later.